internal/envoy/listener.go

```diff
@@ -28,6 +28,18 @@ import (
 	"github.com/heptio/contour/internal/dag"
 )
 
```

Comment:
Rather than documenting where this is used or what it used for, comment on _why_ 60 seconds is the number that was chosen. 

Documenting how a public constant can be used is not possible because anyone can import and use that value without you even knowing.